app/controllers/assets_controller.rb

```diff
@@ -34,15 +33,22 @@ def file_preview
                elsif @assoc.class == RepositoryCell
                  can_manage_repository_rows?(@repository.team)
                end
-
-    if @asset.image?
-      if ['image/jpeg', 'image/pjpeg'].include? @asset.content_type
+    if response_json['type'] == 'image'
+      if ['image/jpeg', 'image/pjpeg'].include? @asset.file.content_type
```

Comment:
Why do we need this double check here? I thing .image? already covers it

Reply:
Only jpeg has quality parameter